resources/user-agents/apps/microsoft/bing-search-app-ios-6.json

```diff
@@ -0,0 +1,46 @@
+{
+  "division": "Bing Search App #MAJORVER#.#MINORVER# for iOS",
+  "versions": ["6.11", "6.9", "6.8", "6.7", "6.6", "6.5", "6.4", "6.3", "6.1"],
+  "sortIndex": 487,
+  "lite": false,
+  "standard": true,
+  "userAgents": [
+    {
+      "userAgent": "Bing Search App #MAJORVER#.#MINORVER# for iOS",
+      "platform": "iOS",
+      "engine": "WebKit",
+      "device": "general Mobile Device (Apple)",
+      "properties": {
+        "Parent": "DefaultProperties",
+        "Comment": "Bing Search App #MAJORVER#.#MINORVER# for iOS",
+        "Browser": "Bing Search App",
+        "Browser_Type": "Application",
+        "Browser_Maker": "Microsoft Corporation",
+        "Version": "#MAJORVER#.#MINORVER#",
+        "Crawler": "false"
+      },
+      "children": [
+        {
+          "match": "Mozilla/5.0 (#DEVICE##PLATFORM#) applewebkit* (*khtml*like*gecko*) Version/* Mobile* BingWeb/#MAJORVER#.#MINORVER#*",
+          "devices": {
+            "iPhone": "iPhone"
+          },
+          "platforms": [
+            "iOS_A_10_3", "iOS_A_10_2",
+            "iOS_A"
+          ]
+        },
+        {
+          "match": "Mozilla/5.0 (#DEVICE##PLATFORM#) applewebkit* (*khtml*like*gecko*) Mobile* BingWeb/#MAJORVER#.#MINORVER#*",
+          "devices": {
+            "iPhone": "iPhone"
+          },
+          "platforms": [
+            "iOS_A_10_3", "iOS_A_10_2",
+            "iOS_A"
+          ]
```

Comment:
Usually we add a pattern that doesn't have a device (and defaults to the `general Mobile Device (Apple)` device) that just has `iOS_A` and `iOS_C` (like this one: https://github.com/browscap/browscap/blob/master/resources/user-agents/browsers/chrome/chrome-ios-48-on.json#L49).

I believe this is mainly done for the Standard file, since it doesn't have the iPhone device available.

Any reason you're not doing that in these files (applies to all of the iOS ones)?

Reply:
changed

Reply:
Ah, I didn't mean remove `iOS_A` from the patterns with the `iPhone` device, but rather have a device-less pattern (for the Standard file), like so:

```json
         {
           "match": "Mozilla/5.0 (#PLATFORM#) applewebkit* (*khtml*like*gecko*) Version/* Mobile* BingWeb/#MAJORVER#.#MINORVER#*",
           "platforms": [
            "iOS_A"
           ]
         },
```

Reply:
added